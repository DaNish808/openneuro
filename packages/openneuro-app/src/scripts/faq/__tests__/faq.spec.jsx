import React from 'react'
import { shallow } from 'enzyme'
import Faq from '../faq'

describe('faq/faq/Faq', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<Faq />)
    expect(wrapper).toMatchSnapshot()
  })
})
