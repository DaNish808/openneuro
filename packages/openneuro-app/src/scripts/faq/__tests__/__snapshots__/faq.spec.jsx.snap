// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`faq/faq/Faq renders successfully 1`] = `
<div
  className="container faqs"
>
  <h1>
    FAQ's
  </h1>
  <div
    className="panel"
    key="0"
  >
    <div
      className="panel-heading"
    >
      <span>
        How do I get started?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Check out our
         
        <a
          href="https://www.youtube.com/playlist?list=PLU-IMZq18nDsf6VpCrLezLgylfAbRPPV8"
        >
          video tutorial
        </a>
        .
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="1"
  >
    <div
      className="panel-heading"
    >
      <span>
        Is this service free to use?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Yes!
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="2"
  >
    <div
      className="panel-heading"
    >
      <span>
        Can I restrict access to my data and analysis results on OpenNeuro?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Yes, but only for a period of 36 months counted from first successful analysis of data from more than one participant. After this period the dataset and related analysis results will be made publicly available.
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="3"
  >
    <div
      className="panel-heading"
    >
      <span>
        Are there any restrictions on the uploaded data?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Yes. By uploading this dataset to OpenNeuro you have to agree to the following conditions:
        <ul>
          <li>
            you are the owner of this dataset and have any necessary ethics permissions to share the data publicly.
          </li>
          <li>
            This dataset does not include any identifiable personal health information as defined by the Health Insurance Portability and Accountability Act of 1996 (including names, zip codes, dates of birth, acquisition dates, facial features on structural scans etc.).
          </li>
          <li>
            You agree that this dataset and results of all analyses performed on it using the OpenNeuro platform will become publicly available under a Creative Commons CC0 or CC-BY license after a grace period of 36 months counted from first successful analysis of data from more than one participant.
          </li>
        </ul>
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="4"
  >
    <div
      className="panel-heading"
    >
      <span>
        What if I will not be able to publish my paper in 36 months?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        You can apply for up to two 6 month long extensions of the grace period. To apply please contact support. We encourage you to publish a preprint of your work to reduce the uncertainty of the publishing pipeline.
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="5"
  >
    <div
      className="panel-heading"
    >
      <span>
        Can I upload and analyze a publicly available dataset from another repository?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        The mission of OpenNeuro is to incentivize data sharing via providing data analysis service. Thus we discourage using it to analyze already publicly available datasets. This policy might change in the future depending on improvements in analysis cost.
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="6"
  >
    <div
      className="panel-heading"
    >
      <span>
        Do I need to format my data in some special way before uploading it to OpenNeuro?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Yes! OpenNeuro only accepts data in the Brain Imaging Data Structure (BIDS) format. You can read about it more at
         
        <a
          href="http://bids.neuroimaging.io/"
          rel="noopener noreferrer"
          target="_blank"
        >
          bids.neuroimaging.io
        </a>
        . If you have any questions about organizing your data please post them at
        <a
          href="https://neurostars.org/tags/bids"
          rel="noopener noreferrer"
          target="_blank"
        >
          neurostars.org
        </a>
        .
      </span>
    </div>
  </div>
  <div
    className="panel"
    key="7"
  >
    <div
      className="panel-heading"
    >
      <span>
        Do I need to remove facial features from structural images before uploading the data?
      </span>
    </div>
    <div
      className="panel-body"
    >
      <span>
        Yes. We recommend using
         
        <a
          href="https://pypi.python.org/pypi/pydeface"
          rel="noopener noreferrer"
          target="_blank"
        >
          pydeface
        </a>
        . Defacing is strongly preffered over skullstripping, because the process is more robust and yields lower chance of accidentally removing brain tissue.
      </span>
    </div>
  </div>
</div>
`;
